cmd/horcrux/cmd/config.go

```diff
@@ -162,6 +168,7 @@ func initCmd() *cobra.Command {
 		"(i.e. \"tcp://node-1:2222|2,tcp://node-2:2222|3\")")
 	cmd.Flags().IntP("threshold", "t", 0, "indicate number of signatures required for threshold signature")
 	cmd.Flags().StringP("listen", "l", "", "listen address of the signer")
+	cmd.Flags().StringP("debuglisten", "d", "", "listen address for Debug and Prometheus metrics")
```

Comment:
I saw we enable the debug server by default. I would imagine most operators would want this functionality. So we make it opt out. A default will also show the expected string format: e.g. `:8543` or `localhost:8543`

Reply:
Added example as suggested